spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/DeferredLog.java

```diff
@@ -232,7 +232,6 @@ private static void logTo(Log log, LogLevel level, Object message, Throwable thr
 			return;
 		case FATAL:
 			log.fatal(message, throwable);
-			return;
```

Comment:
I'd prefer to leave this in for consistency with the other statements. The other changes look like nice improvements. Thanks!

Reply:
👌Thank you for code review.